src/test_l1_plat_power.c

```diff
@@ -266,32 +268,32 @@ void test_l1_plat_power_negative_PLAT_TERM (void)
  * |Variation / Step|Description|Test Data|Expected Result|Notes|
  * |:--:|---------|----------|--------------|-----|
  * |01|Call PLAT_INIT() - open interface | | PWRMGR_SUCCESS | Should Pass |
- * |02|Call PLAT_API_SetPowerState() - Loop through and set for all values in PWRMgr_PowerState_t | newState=PWRMGR_POWERSTATE_STANDBY_DEEP_SLEEP | PWRMGR_SUCCESS | Should Pass |
+ * |02|Call PLAT_API_SetPowerState() - Iterate through various power states fetched from the KVP profile | newState=PWRMGR_POWERSTATE_STANDBY_DEEP_SLEEP | PWRMGR_SUCCESS | Should Pass |
```

Comment:
instead of "from KVP profile" change as "from profile file"

Reply:
Done